src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1071,191 +1048,218 @@ k
      * @param aircraft
      */
     canvas_draw_info(cc, aircraft) {
-        if (!aircraft.isVisible()) {
+        if (!aircraft.isVisible() || aircraft.hit) {
             return;
         }
 
-        // TODO: flip the logic here and return early to make code more readable.
-        if (!aircraft.hit) {
-            // Initial Setup
-            cc.save();
+        // Initial Setup
+        cc.save();
 
-            const cs = aircraft.callsign;
-            const paddingLR = 5;
-            // width of datablock (scales to fit callsign)
-            const width = clamp(1, 5.8 * cs.length) + (paddingLR * 2);
-            const width2 = width / 2;
-            // height of datablock
-            const height = 31;
-            const height2 = height / 2;
-            // width of colored bar
-            const bar_width = 3;
-            const bar_width2 = bar_width / 2;
-            const ILS_enabled = aircraft.pilot.hasApproachClearance;
-            const lock_size = height / 3;
-            const lock_offset = lock_size / 8;
-            const pi = Math.PI;
-            const point1 = lock_size - bar_width2;
-            let alt_trend_char = '';
-            const a = point1 - lock_offset;
-            const b = bar_width2;
-            const clipping_mask_angle = Math.atan(b / a);
-            // describes how far around to arc the arms of the ils lock case
-            const pi_slice = pi / 24;
-            let match = false;
-            let almost_match = false;
-
-            // Callsign Matching
-            if (prop.input.callsign.length > 1 && aircraft.matchCallsign(prop.input.callsign.substr(0, prop.input.callsign.length - 1))) {
-                almost_match = true;
-            }
+        const cs = aircraft.callsign;
+        const paddingLR = 5;
+        // width of datablock (scales to fit callsign)
+        const width = clamp(1, 5.8 * cs.length) + (paddingLR * 2);
+        const width2 = width / 2;
```

Comment:
could we use `halfWidth` and `halfHeight` here?  the number in the var doesn't give us much clue about what it actually is.